src/__test__/render.test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Country from '../components/Home';
+
+const country = {
+  country: {
+    Country: 'Nigeria',
+    TotalConfirmed: 100,
+  },
+};
+
+describe('Country', () => {
+  it('should render country', () => {
+    const { getByText } = render(
+      <BrowserRouter>
+        <Country {...country} />
+      </BrowserRouter>,
+    );
+    expect(getByText('Nigeria')).toBeTruthy();
+  });
+});
+
+
+jest.mock('../components/Dashboard', () => () => <div>Mocked Dashboard</div>);
+// describe('Dashboard component ', () => {
+//   it('should render Dashboard component', async () => {
+//     render(
+//       <Provider store={store}>
+//         <Dashboard />
+//       </Provider>,
+//     );
+//   });
+// });
```

Comment:
### Optional
- [ ] It would be better if we remove the commented piece of code, it makes our codebase looks messy